website/docs/test_core_lib.md

```diff
@@ -0,0 +1,231 @@
+---
+id: test_core_lib
+title: Test Core Lib
+sidebar_label: Test Core Lib
+---
+
+Let's understand how `Core-Lib` is initialized and tested and how to integrate it with your new or existing application.
+
+
+## DataAccess
+Here you should define the functions that only and only communicate with the connection created, for e.g., queries to get some data from a database or insert some data in the database, Solr queries or whatever the function has to interface with the connection.
+
+`user_data_access.py`
+```python
+from http import HTTPStatus
+
+from core_lib.data_layers.data_access.db.crud.crud_data_access import CRUDDataAccess
+from core_lib.connection.sql_alchemy_connection_registry import SqlAlchemyConnectionRegistry
+from core_lib.error_handling.status_code_exception import StatusCodeException
+from user_core_lib.data_layers.data.db.user import User
+from core_lib.error_handling.not_found_decorator import NotFoundErrorHandler
+
+
+class UserDataAccess(CRUDDataAccess):
+    def __init__(self, db: SqlAlchemyConnectionRegistry):
+        CRUD.__init__(self, User, db)
+```
+
+## Service 
+Here we right the business logic for the extracted data, as the `DataAccess` only handles the interfacing with the connection, a `Service` will only be used to take the data and process it.
```

Comment:
suggestion:
The `Service` layer is a facade of the `DataAccess` layer and connections. consisting of `API` functions that will handle business logic, data transformation, and caching 

maybe also add to `1.data_layers.md`?